source/braille.py

```diff
@@ -2306,10 +2303,12 @@ def start(cls):
 
 	@classmethod
 	def queueApc(cls, func, param=0):
+		# Ensure the thread is running
+		cls.start()
 		ctypes.windll.kernel32.QueueUserAPC(func, cls.handle, param)
 
 	@classmethod
-	def stop(cls, timeout=None):
+	def stop(cls):

```

Comment:
dropping this parameter is an API breaking change. I think we should try to preserve backwards compatibility here if possible, even if setting `timeout` has no change of behaviour.

Reply:
I reverted that change.